tests/testerina-integration-test/src/test/java/org/ballerinalang/testerina/test/MockTest.java

```diff
@@ -61,11 +61,93 @@ public void testFunctionMocking() throws BallerinaTestException {
     public void testObjectMocking() throws BallerinaTestException {
         String msg1 = "7 passing";
         String msg2 = "6 failing";
+
+        String msg3 = "incorrect type of argument provided at position '1' to mock the function get()";
+
         String[] args = mergeCoverageArgs(new String[]{"object-mocking-tests"});
         String output = balClient.runMainAndReadStdOut("test", args,
                 new HashMap<>(), projectPath, false);
-        if (!output.contains(msg1) || !output.contains(msg2)) {
+        if (!output.contains(msg1) || !output.contains(msg2) || !output.contains(msg3)) {
             Assert.fail("Test failed due to object mocking failure in test framework.\nOutput:\n" + output);
         }
     }
+
+    @Test()
+    public void testObjectMocking_DefaultIncompatibleArgs() throws BallerinaTestException {
+        String msg = "incorrect type of argument provided at position '1' to mock the function get()";
```

Comment:
```suggestion
        String msg = "incorrect type of argument provided at position '1' to mock the function 'get()'";
```

Reply:
To have this change, Ill have to modify the message in the `ObjectMock` class 

https://github.com/ballerina-platform/ballerina-lang/blob/master/misc/testerina/modules/testerina-core/src/main/java/org/ballerinalang/testerina/natives/mock/ObjectMock.java#L180

Ill proceed with applying this wherever applicable

Reply:
Applied single quotes where applicable